### PR TITLE
Game over dialog

### LINKED ISSUE #72: Game losing terms
- add game losing terms - population equal to 0

- add info to show user that they lost the game:
- - how many waves did they survived 
- - game time

- move to home page screen

### PR DESCRIPTION
**Pull request by bartoszbialecki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# village-defense
+# village-defence
```
Comment (pdybowski) [inline README.md:1]: doesnt matter. both are correct xd

File: src/app/components/gameOverDialog.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { Game } from '../game'
+import { createElement, secondsToDate } from '../utils'
+import { renderPopup } from './popup'
+
+export default class GameOverDialog {
```
Comment (pdybowski) [inline src/app/components/gameOverDialog.ts]: we can use Dialog or Modal, in project I changed every popup to Modal. You can change them to Dialogs or change this on to gameOverModal

File: src/app/screens/gamepage/index.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import { App } from '../../app'
+import GameOverDialog from '../../components/gameOverDialog'
```
Comment (pdybowski) [inline src/app/screens/gamepage/index.ts]: from '../../components'

**Review by hoolek77 — APPROVED** [review]

**Review by pdybowski — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bartoszbialecki) [reply README.md:1]: Maybe, but we should use one version everywhere, which one, it doesn't matter.

**Review by pdybowski — APPROVED** [review]

**Pull request merged**